core/Dockerfile

```diff
@@ -64,6 +64,9 @@ ENV TARGETVARIANT=${TARGETVARIANT}
 COPY tools /home/pi/tools
 RUN /home/pi/tools/install-system-tools.sh
 
+# Copy overlay source in case we need to build it in runtime
+COPY install/ /install
```

Comment:
Since etc/blueos is already being binded to the host, I believe that best place would be /usr/share/blueos/install 
@joaoantoniocardoso

Reply:
Since `install` is a script, I think it's more likely for it to find a comfy home as `/opt/blueos/install`

Reply:
> Since `install` is a script, I think it's more likely for it to find a comfy home as `/opt/blueos/install`. `/opt/blueos` is a general good home for all sorts of little creatures :)

in this case it is as folder with the .dts file in it...

plus my largerst concern is where it should live in the git repo, as this (/core/install/spi0-led.dts) is a redundant copy of the one at the /install/overlays/spi0-led.dts